src/main/resources/processes/PSU/PSU.bpmn

```diff
@@ -57,55 +57,66 @@
     <bpmn:sequenceFlow id="Flow_0v13kf6" sourceRef="Gateway_1s86vr7" targetRef="EndEvent_PSU">
       <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">${execution.getVariable("ReturnCase") == true || execution.getVariable("PsuTriageOutcome") == "CloseCase"}</bpmn:conditionExpression>
     </bpmn:sequenceFlow>
+    <bpmn:serviceTask id="Service_UpdatePsuDeadline" name="Update Case Deadline" camunda:expression="${bpmnService.updateDeadlineDays(execution.getVariable(&#34;CaseUUID&#34;), execution.processBusinessKey, 60)}">
```

Comment:
This action needs to go either in the registration or triage diagrams. This diagram is generic and is not specific to IEDET and will be used by BF and UKVI which will most likely have different deadlines

Reply:
Code review comments has been incorporated.